src/commands/image/imageSource/buildImageInAzure/BuildImageStep.ts

```diff
@@ -63,11 +72,26 @@ export class BuildImageStep extends ExecuteActivityOutputStepBase<BuildImageInAz
     }
 
     protected createFailOutput(context: BuildImageInAzureImageSourceContext): ExecuteActivityOutput {
+        let loadMoreChildrenImpl: (() => Promise<AzExtTreeItem[]>) | undefined;
```

Comment:
nit: You could define `loadMoreChildrenImpl` here as `Promise.resolve([]))`. That way on line 94, you wouldn't have to do that check and you could type define `loadMoreChildrenImpl` as just `(() => Promise<AzExtTreeItem[]>)`